### ISSUE TITLE
Broken `npm install` under src/parser

### ISSUE THREAD

**Issue by popham** [facebook/flow#387]
I'm running `npm`'s version 1.4.14, and `npm install` runs the `prepublish` hook (see https://github.com/npm/npm/issues/3059).  The `prepublish` hook is currently broken because of a `diff README.md README-npm.md`.  I'm removing the diff from my local copy to get things working (I prefer `npm install && npm link` to `npm install -g flow-parser`).

What is the intention here?  Is `README-npm.md` the readme, is `README.md` the readme, or are there supposed to be two distinct readmes?

**Comment by gabelevi** [comment]
To be clear, the problem is when you `cd src/parser && npm install`, right?

The problem is that there's a [readme for npm](npmjs.com/package/flow-parser) and a [readme for github](https://github.com/facebook/flow/tree/master/src/parser). Unfortunately, as far as I can tell, both npm and github insist on using `README.md` and cannot be configured to read a different file.

At first I tried to make prepublish move the `README`'s around and postpublish fix them, but this didn't work because
* npm seems to read the `README.md` before prepublish is run
* prepublish is run on npm install

So I got rid of that. At the last minute I added the `diff` command to stop someone from publishing with the wrong `README` but as you noted, that breaks `npm install`. I've removed the `diff` command from the internal version which will be deployed later this week.
